generator/code_generator.go

```diff
@@ -683,9 +678,7 @@ func (s *Service) setLogValueByMessageArgument(msg *resolver.Message) {
 			Key:   field.Name,
 			Value: s.msgArgumentLogValue(field),
 		})
-		if field.Type.Ref != nil {
```

Comment:
IIUC, this is the part where some of the log values are missing?

Reply:
Yes !